src/main/java/seedu/address/logic/commands/findcommand/FindTeamCommand.java

```diff
@@ -54,4 +58,18 @@ public CommandResult execute(Model model) {
         model.updateHistory(this);
         return new CommandResult(MESSAGE_SUCCESS, CommandType.T);
     }
+
+    @Override
```

Comment:
Hey, I'm not sure whether it's because it's not merge in, I tried to pull from the master branch and use 'find'. Would there be errors thrown if the inputs can't be found? :bowtie:

Reply:
yea thats odd. it should throw error